File: app/src/main/java/com/example/sample/sample/myapplication/WebViewActivity.java
package com.example.sample.sample.myapplication;

import android.app.Activity;
import android.os.Bundle;

import com.example.lib.effect.effect.widget.SpringRelativeLayout;
import com.example.lib.effect.effect.widget.SpringWebView;
import com.example.sample.myapplication.R;


public class WebViewActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.spring_webview);

        SpringRelativeLayout springRelativeLayout = (SpringRelativeLayout) findViewById(R.id.spring_layout);
        springRelativeLayout.addSpringView(R.id.web_view);
        SpringWebView webview = (SpringWebView) findViewById(R.id.web_view);
        String url = "file:///android_asset/copyright_en.htm";
        webview.getSettings().setBuiltInZoomControls(false);
        webview.loadUrl(url);

        //webview.getSettings().setJavaScriptEnabled(true);
        //webView.setWebViewClient(new WebViewClient()); //不調用系統瀏覽器
        //webview.loadUrl("https://www.qq.com/");

        webview.setEdgeEffectFactory(springRelativeLayout.createViewEdgeEffectFactory());
    }
}

File: app/src/main/java/com/example/sample/sample/myapplication/WebViewActivity.kt
package com.example.sample.sample.myapplication

import android.app.Activity
import android.os.Bundle
import android.view.View
import com.example.lib.effect.effect.widget.SpringRelativeLayout
import com.example.lib.effect.effect.widget.SpringWebView
import com.example.sample.myapplication.R

class WebViewActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.spring_webview)
        val springRelativeLayout = findViewById<View>(R.id.spring_layout) as SpringRelativeLayout
        springRelativeLayout.addSpringView(R.id.web_view)
        val webview = findViewById<View>(R.id.web_view) as SpringWebView
        val url = "file:///android_asset/copyright_en.htm"
        webview.settings.builtInZoomControls = false
        webview.loadUrl(url)

        //webview.getSettings().setJavaScriptEnabled(true);
        //webView.setWebViewClient(new WebViewClient()); //not Android System WebView
        //webview.loadUrl("https://www.amazon.com/");
        webview.setEdgeEffectFactory(springRelativeLayout.createViewEdgeEffectFactory())
    }
}